src/ServicePulse.Host/app/modules/monitoring/js/endpoint_details.controller.js

```diff
@@ -99,7 +111,8 @@
         'serviceControlService',
         'monitoringService',
         'historyPeriods',
-        'formatter'
+        'formatter',
```

Comment:
Do we still need the formatter to be injected?

Reply:
No, I think it can be removed, let me do this